Microsoft.DotNet.Try.Jupyter/IMessageBuilder.cs

```diff
@@ -1,21 +1,41 @@
 // Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
 using System.Collections.Generic;
 using Microsoft.DotNet.Try.Jupyter.Protocol;
 
 namespace Microsoft.DotNet.Try.Jupyter
 {
     public interface IMessageBuilder
     {
-        Header CreateHeader(string messageType, string session);
+        Message CreateMessage(JupyterMessageContent content, Header parentHeader, List<byte[]> identifiers = null);
```

Comment:
Does this need to be a list? `ReadOnlySpan<byte>` maybe?

Reply:
is a list of byte []

Reply:
Meaning it needs to be mutable?